extensions/adobe/experience/decisioning/ranking.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/decisioning/ranking",
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/decisioning/option"
+  ],
+  "meta:abstract": true,
+  "meta:extensible": true,
+  "type": "object",
+  "title": "Decision Option Ranking",
+  "description": "A ranking produces the order in which one option is selected over another. A fixed absolute priority can be used in case there is no other function known that maps a decsion option to an ordinal value.",
+  "definitions": {
+    "ranking": {
+      "properties": {
+        "https://ns.adobe.com/experience/decisioning/ranking": {
+          "$ref": "https://ns.adobe.com/experience/decisioning/ranking-details",
+          "title": "Ranking Details",
+          "description": "Rank (priority). Defines what is considered the \"best action\" given the context of the decision criterion. Among all the selected options that meet the eligibility constraint, the ranking order will decide the top (or top N) option(s) to be proposed."
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/ranking"
+    }
+  ],
+  "meta:status": "experimental"
```

Comment:
Does this need to be "experimental" ?

Reply:
Yes, it fits the definition https://github.com/adobe/xdm/blob/master/docs/status.md 
"stabilizing" would also work but its use is discouraged

Reply:
Please see the previous comment

Reply:
I updated the meta:status to allow all orgs to see the schemas. We have workflows that can't distinguish between Beta orgs and everyone else. The schemas will be needed for all production orgs.